### DISCUSSION TITLE
Error in node js

### DISCUSSION THREAD

**Discussion by touqeer786** [community/community#21433] [Other Feature Feedback, Questions, & Ideas]
<p>npm ERR! cb() never called!</p>
<p>npm ERR! This is an error with npm itself. Please report this error at:<br>
npm ERR!     <a href="https://npm.community" rel="noopener nofollow ugc">https://npm.community</a></p>
<p>npm ERR! A complete log of this run can be found in:<br>
npm ERR!     C:\Users\atouq\AppData\Roaming\npm-cache_logs\2021-09-23T17_38_29_063Z-debug.log</p>
<p>Aborting installation.</p>

**Comment by nazimboudeffa** [comment]
<p>Hello,</p>
<p>Is this error related to GitHub ? Maybe you can ask on <a href="https://stackoverflow.com" rel="noopener nofollow ugc">stackoverflow.com</a></p>

**Comment by Zerotask** [comment, marked as answer]
<p>It seems that the linked URL points to this forum, but Stackoverflow is indeed a better place for it.<br>
Alternatively, if you think this is a bug to npm itself, they also have a repository on GitHub: <a href="https://github.com/npm/cli" class="inline-onebox" rel="noopener nofollow ugc">GitHub - npm/cli: the package manager for JavaScript</a></p>
